ftp/src/main/scala/akka/stream/alpakka/ftp/SFtp.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.ftp
+
+import akka.stream.alpakka.ftp.FtpConnectionSettings.SshBasicFtpConnectionSettings
+import com.jcraft.jsch.{ ChannelSftp, JSch }
+import scala.collection.immutable
+import scala.util.Try
+
+/**
+ * @author Juan José Vázquez Delgado
+ */
+trait SFtp {
```

Comment:
think about what classes should be public and what is internal. Keep the public API as small as possible. Mark internal things with `private[alpakka]`, perhaps move to an `impl` package also.